src/telegram/ext/_application.py

```diff
@@ -1256,8 +1256,14 @@ async def process_update(self, update: object) -> None:
         context = None
         any_blocking = False  # Flag which is set to True if any handler specifies block=True
 
-        for handlers in self.handlers.values():
+        # We "deepcopy" the dict to avoid issues with concurrent modification of the
```

Comment:
```suggestion
        # We copy the list to avoid issues with concurrent modification of the
```
a bit confusing with the comment just below it `# do *not* use copy.deepcopy ..`